travvy-server/src/controllers/RecommendController.js

```diff
@@ -40,10 +41,10 @@ module.exports = {
     return attractions
     }
     else{
-      const activity_ids = await Users_type.findAll({
+      const activity_ids = await Users.findAll({
         attributes: ['activity_id'],
-        model: Users_type,
-        as: 'Users_type',
+        model: Users,
+        as: 'Users',
         where: {Email: user} 
```

Comment:
Thanks for applying the changes in the recommend controller as well, are the unit tests still passing for the recommend controller?

Reply:
No problem! There is one that is failing - I have a note to bring this up in our stand-up tomorrow. The test that is failing is called "Recommend Attractions User with Preferences Test", which is now being tested in the unit test that I created. This should be able to be deleted from your unit tests from the home page - but we can discuss more tomorrow and edit the unit test file.

Reply:
The reason it was failing is because the unit test is using "dyampolsky@uwaterloo.ca" as the user but in the database there was no activity_id for the user. So I set the value in the database it should be passing the test now, it is for me.

Reply:
Oh great, thanks for looking into this more Daria. I will test it out as well.